client/src/index.js

```diff
@@ -3,9 +3,9 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
-import DetailedCard from './components/DetailedCard/DetailedCard';
+import CardCheckList from './components/CardCheckList/CardCheckList';
 
-ReactDOM.render(<DetailedCard />, document.getElementById('root'));
+ReactDOM.render(<CardCheckList />, document.getElementById('root'));
```

Comment:
This is overwriting the whole App, let's use the App component here please. Then I'll approve!